pkg/providers/launchtemplate/suite_test.go

```diff
@@ -2220,6 +2220,45 @@ essential = true
 			})
 		})
 	})
+	Context("Networking", func() {
```

Comment:
Each of these tests sets the ClusterIPFamily but we added logic to generate the value for this member -- do we need to expand this test/add a new test that validates that we are setting this value to the right thing?

Reply:
Yeah, makes sense. Let me add it as well.